src/TableRenderers.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, {Fragment} from 'react';
```

Comment:
```suggestion
import React from 'react';
```

No need to import `Fragment` explicitly since you're using it this way `<React.Fragment>`